src/components/common/input/__snapshots__/input.test.js.snap

```diff
@@ -14,8 +14,8 @@ exports[`<Input /> Rendering should render the component 1`] = `
 exports[`<Input /> Rendering should render the component with a proper placeholder  1`] = `
 <input
   className="search-input"
-  onInput={[MockFunction]}
-  onKeyPress={[MockFunction]}
+  onInput={[Function]}
+  onKeyPress={[Function]}
```

Comment:
Why these functions are not mocks now?

Reply:
the test used wrong props for the Input component. Fixed